release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -511,6 +511,33 @@ $ oc get packagemanifests <operator_name> -n <catalog_namespace> -o <output_form
 
 For more information, see xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-installing-specific-version-cli_olm-adding-operators-to-a-cluster[Installing a specific version of an Operator].
 
+[id=ocp-4-13-olm-multitenant]
+==== Operators in multitenant clusters
+
+The default behavior for Operator Lifecycle Manager (OLM) aims to provide simplicity during Operator installation. However, this behavior can lack flexibility, especially in multitenant clusters.
+
+Guidance and a recommended solution for Operator management in multitenant clusters has been added with the following topics:
+
+* xref:../operators/understanding/olm-multitenancy.adoc#olm-multitenancy[Operators in multitenant clusters]
+* xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-preparing-operators-multitenant_olm-adding-operators-to-a-cluster[Preparing for multiple instances of an Operator for multitenant clusters]
+
+[id=ocp-4-13-olm-colocation-namespace]
+==== Colocation of Operators in a namespace
+
+Operator Lifecycle Manager (OLM) handles OLM-managed Operators that are installed in the same namespace, meaning their Subscription resources are colocated in the same namespace, as related Operators. Even if they are not actually related, OLM considers their states, such as their version and update policy, when any one of them is updated.
+
+Guidance on Operator colocation and an alternative procedure that uses custom namespaces has been added with the following topics:
+
+* xref:../operators/understanding/olm/olm-understanding-dependency-resolution.adoc#olm-colocation-namespaces_olm-understanding-dependency-resolution[Colocation of Operators in a namespace]
+* xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-installing-global-namespaces_olm-adding-operators-to-a-cluster[Installing global Operators in custom namespaces]
+
+[id=ocp-4-13-olm-disabled-csvs]
+==== Updated web console behavior when disabling copied CSVs
+
+The {product-title} web console has been updated to provide better Operator discovery copied cluster service versions (CSVs) are disabled on a cluster. When copied CSVs are disabled by a cluster administrator, the web console is modified to show copied CSVs from the `openshift` namespace in every namespace for regular users, even though the CSVs are not actually copied to every namespace. This allows regular users to still be able to view the details of these Operators in their namespaces and create custom resources (CRs) brought in by globally installed Operators.
```

Comment:
> The {product-title} web console has been updated to provide better Operator discovery copied cluster service versions (CSVs) are disabled on a cluster.

I guess this sentence lacks `when`, it should be I guess:
The {product-title} web console has been updated to provide better Operator discovery **when** copied cluster service versions (CSVs) are disabled on a cluster.

Reply:
Oops yes, thank you!